### ISSUE TITLE
About the utils/imutils.py line:41

### ISSUE THREAD

**Issue by lucas946** [GengDavid/pytorch-cpn#29]
I think the following code is confusing or not correct,
   heatmap /= am / 255
because batchnorm is the last layer of the predict net, the single element of heatmap should within the range of 0-1. I think the code should be correct as followed,
  heatmap /= am.
But I'm not totally sure I am right, can you explain it?

**Comment by GengDavid** [comment]
Hope [this](https://github.com/chenyilun95/tf-cpn/issues/32#issuecomment-394617506) would help you to understand.
